runtime/runtime/src/prefetch.rs

```diff
@@ -91,30 +91,36 @@ impl TriePrefetcher {
         receipts: &[Receipt],
     ) -> Result<(), PrefetchError> {
         for receipt in receipts.iter() {
-            if let Some(action_receipt) = &receipt.receipt.action() {
-                let account_id = receipt.receiver_id.clone();
+            match &receipt.receipt {
+                ReceiptEnum::Action(action_receipt) | ReceiptEnum::PromiseYield(action_receipt) => {
```

Comment:
FYI: When applying a similar pattern in the future (matching on something with a divergent default case,) `let-else` may help to write this more concisely and with less right-ward drift:

```rust
let (ReceiptEnum::Action(r) | ReceiptEnum::PromiseYield(r)) = &receipt.receipt else {
    continue;
};
let action_receipt = r;
use(action_receipt);
```

though in this case `match` makes it easier to fit the pattern into 100 column limit without making a compromise in formatting or introducing single letter variables.